ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,225 @@
+
+    def main(self, *, args):
+
+        maintainer_name = getpass.getuser()
+        if not args.maintainer_name:
+            # try getting the name from the global git config
+            p = subprocess.Popen(
+                ['git', 'config', '--global', 'user.name'],
+                stdout=PIPE)
+            resp = p.communicate()
+            name = resp[0].decode().rstrip()
+            if name:
+                maintainer_name = name
+        else:
+            maintainer_name = args.maintainer_name
+
+        maintainer_email = maintainer_name + '@todo.todo'
+        if not args.maintainer_email:
+            # try getting the email from the global git config
+            p = subprocess.Popen(
+                ['git', 'config', '--global', 'user.email'],
+                stdout=PIPE)
+            resp = p.communicate()
+            email = resp[0].decode().rstrip()
+            if email:
+                maintainer_email = email
+        else:
+            maintainer_email = args.maintainer_email
+
+        if args.cpp_node_name == args.cpp_library_name:
+            raise ValueError('cpp_node_name has to be different from cpp_library_name')
+
+        print('going to create a new package')
+        print('package name:', args.package_name)
+        print('destination directory', args.destination_directory)
+        print('build tool:', args.build_tool)
+        print('maintainer_email:', maintainer_email)
+        print('maintainer_name:', maintainer_name)
+        print('dependencies:', args.dependencies)
+        print('cpp_node_name:', args.cpp_node_name)
+        print('cpp_library_name:', args.cpp_library_name)
+
+        package_directory = create_folder(args.package_name, args.destination_directory)
+        if not package_directory:
+            print('unable to create folder', args.destination_directory)
+            return False
+
+        package_xml_config = {
+            'package_name': args.package_name,
+            'maintainer_email': maintainer_email,
+            'maintainer_name': maintainer_name,
+            'dependencies': args.dependencies,
+        }
+
+        if args.build_tool == 'cmake':
+            create_template_file(
+                'cmake/package.xml.em',
+                package_directory,
+                'package.xml',
+                package_xml_config)
+
+            cmakelists_config = {
+                'project_name': args.package_name,
+                'dependencies': args.dependencies,
+                'cpp_node_name': args.cpp_node_name,
+                'cpp_library_name': args.cpp_library_name,
+            }
+            create_template_file(
+                'cmake/CMakeLists.txt.em',
+                package_directory,
+                'CMakeLists.txt',
+                cmakelists_config)
+
+            cmake_config = {
+                'project_name': args.package_name,
+                'create_cpp_library': True if args.cpp_library_name else False,
```

Comment:
The expression can be simplified to: `bool(args.cpp_library_name)`.

Same below.